src/renderers/shared/fiber/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -935,4 +935,176 @@ describe('ReactIncrementalErrorHandling', () => {
     });
     expect(() => ReactNoop.flush()).toThrow('Error!');
   });
+
+  describe('ReactFiberErrorLogger', () => {
+    function initReactFiberErrorLoggerMock(mock) {
+      jest.resetModules();
+      if (mock) {
```

Comment:
This is awkward. I'm not sure how to test both _unmocked_ and _mocked_ without explicitly doing this.